.travis.yml

```diff
@@ -1,6 +1,7 @@
+sudo: false
 language: node_js
 node_js:
-- 'node'
+- '10'
```

Comment:
what's the purpose of this?  shouldn't we always run on the latest?

Reply:
I couldn't find which node version everyone was running this on. I think it's better if we upgrade it manually rather than letting it be automatic. So, if 12 comes out, then we can replace this with 12 and then say that the app runs on 12.